source/extensions/filters/http/jwt_authn/jwt_cache.cc

```diff
@@ -56,7 +60,10 @@ class JwtCacheImpl : public JwtCache {
   }
 
   void insert(const std::string& token, std::unique_ptr<::google::jwt_verify::Jwt>&& jwt) override {
-    if (jwt_lru_cache_ && token.size() <= kMaxJwtSizeForCache) {
+    if (!jwt_lru_cache_ || token.size() > std::numeric_limits<uint32_t>::max()) {
+      return;
+    }
+    if (static_cast<unsigned int>(token.size()) <= max_jwt_size_for_cache_) {
```

Comment:
```suggestion
    if (static_cast<uint32_t>(token.size()) <= max_jwt_size_for_cache_) {
```

Just in case on some architecture `unsigned int` is not the same as uint32_t

Reply:
Thanks, I forgot about it. Done.